spark/v3.2/build.gradle

```diff
@@ -33,6 +33,8 @@ configure(sparkProjects) {
       resolutionStrategy {
         force 'com.fasterxml.jackson.module:jackson-module-scala_2.12:2.12.3'
         force 'com.fasterxml.jackson.module:jackson-module-paranamer:2.12.3'
+        // enforce the newer version of netty buffer in test
+        force 'io.netty:netty-buffer:4.1.68.Final'
```

Comment:
Is this needed if the dependency is excluded from the 3.2 Spark dependency?

Reply:
Yes. Netty buffer can come from project(':iceberg-arrow') if we don't enforce here.

Reply:
Isn't the point to use the same version as the one from iceberg-arrow?

Reply:
We cannot use it for Spark 3.2, which cause the issue #3405. Maybe we can let iceberg-arrow use the newer version of netty buffer(4.1.63+), so they are consistent?

Reply:
Yeah, let's try updating the version in iceberg-arrow instead of forcing it here. Always best to do the build with as few versions as possible.

Reply:
Iceberg-arrow depends on the lib arrow-memory-netty, which brings in netty buffer. 
https://github.com/apache/iceberg/blob/7f520b0c54fbe20a8fd746697bbcba555621c242/build.gradle#L489
After #3040, we got arrow-memory-netty 5.0, which still depends on an older netty buffer version(4.1.48.Final). We need at least 4.1.63.Final for Spark3.2. Here are two options:
- Option 1, to bump up the netty buffer to 4.1.68.Final, which may or may not break the arrow-memory-netty.
- Option 2, to bump up the arrow-memory-netty to 6.0, which depends on a newer version(4.1.68) of netty buffer, like what we did in #3040. It is a bigger task, need more test and people familiar with Arrow to review.

Need your thoughts. @rdblue @aokolnychyi @kbendick @nastra

Reply:
I think we should bump the version to 4.1.63.Final. There's less risk doing that and we don't need to build Arrow against a different version than what will be used in Spark.

Reply:
Made the change. Spark 3.2 Jmh works now, and verified that the runtime jar got the shaded netty buffer with version 4.1.63.Final.